source/common/grpc/common.cc

```diff
@@ -53,8 +53,8 @@ void Common::chargeStat(const Upstream::ClusterInfo& cluster, const std::string&
   chargeStat(cluster, "grpc", grpc_service, grpc_method, success);
 }
 
-Optional<Status::GrpcStatus> Common::getGrpcStatus(const Http::HeaderMap& trailers) {
```

Comment:
This is a weird one. Why doesn't clang-tidy like this? Semantically, the gRPC status comes from the trailers.